### DISCUSSION TITLE
[Feature] Sass support

### DISCUSSION THREAD

**Discussion by silasbispo** [remix-run/remix#5432] [Proposals]
### What is the new or updated feature that you are suggesting?
Support sass natively, been possible to importing in `Links` component and ready to production
### Why should this feature be included?
Because all frameworks like Vue, React, Next, Nuxt all theses, support you to import sass files and style.

I know i can transpile it on my own, but having duplicated pages and files with same style content one in .scss/ and other .css is not the better approach possible

**Comment by Ryongyon** [comment]
I've been looking forward to this feature since 2021. At present, the official support for built-in PostCSS and TailwindCss is just beginning. I'm not sure whether the official will consider supporting SCSS. 

I also hope that the official will consider supporting built-in SCSS, like PostCSS and TailwindCss, out of the box, rather than compiling CSS through the CLI and importing it.

**Comment by wingleung** [comment]
The proposed solution of running sass as a separate process alongside remix just isn't scaling very well when the remix project grows.
So I took some time to figure out how we can run SCSS and SCSS modules in remix 
- https://github.com/remix-run/remix/pull/7612

We will start testing this on our project through patching, but it would be good if I can get some feedback on the PoC PR and maybe get it into remix. Especially if the setup is something that fits in remix's future plans 🤞

**Comment by harshilsharma63** [comment]
I was excited to use Remix but 15 mins in and realised it doesn't support SASS/SCSS seamlessly and requires ugly looking workarounds. This is a dealbreaker for me. Moving onto the next framework.

**Reply by oise** [reply]
With Vite, SCSS/SASS works . You need to install Sass and you are good to go.

**Comment by brophdawg11** [comment]
This discussion is being closed as part of the Remix v3 repository cleanup.

Remix v1/v2 was [upstreamed into React Router v7](https://remix.run/blog/wake-up-remix), so if this proposal is still relevant, please reopen or continue the discussion in the [React Router](https://github.com/remix-run/react-router) repository.
